ross/shaft_element.py

```diff
@@ -1192,3 +1192,1133 @@ def section(
         ]
 
         return elements
+
+
+class ShaftElement6DoF(Element):
```

Comment:
Here we should also consider inheriting from `ShaftElement` to simplify the code.